apis/nucleus/src/components/listbox/interactions/keyboard-navigation/keyboard-nav-rows.js

```diff
@@ -75,12 +75,7 @@ export default function getRowsKeyboardNavigation({
         }
 
         // Without shift key
-        if (inSelection) {
-          keyboard.focusSelection();
-        } else {
-          currentTarget.blur();
-          keyboard.blur();
```

Comment:
Could we remove this one without causing problem?

Reply:
Probably not. `keyboard.blur()` tells nebula that the chart (listbox) has been blurred, and thus updates it state. There might be issues when you try to focus the listbox again, since nebula think the listbox still has focus

Reply:
I somehow find it strange that it suddenly get blur instead of going to the objects in side the filter-pane. I think it makes more sense that it is not blur unless user pressing escape but I don't much about filter-pane that change might not be good.